suite-native/bluetooth/src/hooks/useBluetoothScanner.ts

```diff
@@ -0,0 +1,25 @@
+import { useCallback } from 'react';
+import { AppState } from 'react-native';
+
+import { BleError, bluetoothManager } from '@trezor/transport-native-bluetooth';
+
+export const useBluetoothScanner = () => {
+    const startDeviceScan = useCallback((errorHandler?: (error: BleError) => void) => {
+        bluetoothManager.startDeviceScan(errorHandler);
+
+        const subscription = AppState.addEventListener('change', nextAppState => {
+            if (nextAppState === 'active') {
+                bluetoothManager.startDeviceScan(errorHandler);
+            } else {
+                bluetoothManager.stopDeviceScan();
+            }
+        });
+
+        return () => {
+            subscription.remove();
+            bluetoothManager.stopDeviceScan();
+        };
+    }, []);
+
+    return { startDeviceScan };
```

Comment:
What about this name? 
```suggestion
    return { startAndSubscribeDeviceScan };
```

Reply:
I don't understand the motivation... Subscribe to what? Ain't that implementation detail?

Reply:
Yeah, it's probably redundant, just trying to make it more explicit but `startDeviceScan` is probably good enough.